internal/tcp/tcp_test.go

```diff
@@ -21,62 +21,61 @@
 package tcp
 
 import (
-	coreLog "log"
 	"net"
 	"runtime"
 	"testing"
+	"time"
 
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
 	"github.com/uber/arachne/defines"
-	"github.com/uber/arachne/internal/log"
-	"github.com/uber/arachne/internal/util"
 
 	"github.com/stretchr/testify/assert"
-	"go.uber.org/zap"
 )
 
-func TestReceiver(t *testing.T) {
-	var interfaceName string
-	var err error
+func TestParsePktTCP(t *testing.T) {
+	assert := assert.New(t)
+
+	pktData := []byte{69, 0, 0, 55, 145, 204, 64, 0, 56, 6, 81, 145, 10, 1, 1, 10, 10, 0, 0, 10, 121, 24, 172, 79, 118, 141, 174, 112, 251, 246, 199, 17, 80, 18, 170, 170, 154, 237, 0, 0, 1, 0, 0, 0, 14, 208, 222, 40, 104, 36, 169, 82, 116, 0, 0}
```

Comment:
humans can't read this so it should be documented in some way. what is the nature of this packet that makes it suitable for this test.